integrations/extensions/starter-kits/language-model-watsonx/watsonx-actions.json

```diff
@@ -651,36 +651,163 @@
                         "skill_variable": "watsonx_project_id"
                       },
                       "parameter": "project_id"
-                    },
+                    }
+                  ],
+                  "query": [
                     {
                       "value": {
-                        "skill_variable": "model_parameters_temperature"
+                        "skill_variable": "watsonx_api_version"
                       },
-                      "parameter": "parameters.temperature"
-                    },
+                      "parameter": "version"
+                    }
+                  ]
+                },
+                "result_variable": "step_817_result_2"
+              }
+            },
+            "variable": "step_817",
+            "next_step": "step_606"
+          },
+          {
+            "step": "step_606",
+            "output": {
+              "generic": []
+            },
+            "context": {
+              "variables": [
+                {
+                  "value": {
+                    "expression": "${step_817_result_2.body.results}[0].generated_text"
+                  },
+                  "skill_variable": "model_response"
+                }
+              ]
+            },
+            "handlers": [],
+            "resolver": {
+              "type": "end_action"
+            },
+            "variable": "step_606",
+            "condition": {
+              "and": [
+                {
+                  "eq": [
                     {
-                      "value": {
-                        "skill_variable": "model_parameters_max_new_tokens"
-                      },
-                      "parameter": "parameters.max_new_tokens"
+                      "variable": "step_817_result_2",
```

Comment:
This does not work in `Invoke Watson Deployment prompt API` and shows up as missing variable in `Invoke watsonx Generation API` 

![image](https://github.com/watson-developer-cloud/assistant-toolkit/assets/16194561/f8568408-b0be-44c0-a633-c04f71f81017)

Reply:
I haven't tried it out yet, but that typically happens when the actions.json file was exported from an older version of the openapi spec.  So @modcarroll , I would recommend that you set this up again with the latest openapi spec and then re-do the export of the actions.json file.

Reply:
I'm unable to replicate these errors when I re-setup the action. Are either of you available sometime for a quick screenshare?

Reply:
I believe I fixed this issue - can you check the latest commit and check again?